addons/website_slides/static/src/js/slides_course_quiz.js

```diff
@@ -251,6 +260,30 @@ odoo.define('website_slides.quiz', function (require) {
             });
         },
 
+        /**
+         * Will check if we have answers coming from the session and re-apply them.
+         */
+        _checkSessionAnswers: function () {
+            if (!this.slide.answers || this.slide.answers.length === 0) {
+                return;
+            }
+
+            var self = this;
+            this.$('.o_wslides_js_lesson_quiz_question').each(function () {
+                var $question = $(this);
+                $question.find('a.o_wslides_quiz_answer').each(function () {
+                    var $answer = $(this);
+                    if (!$answer.find('input[type=radio]')[0].checked &&
+                        _.contains(self.slide.answers, $answer.data('answerId'))) {
+                        $answer.find('input[type=radio]').prop('checked', true);
+                    }
+                });
+            });
+
+            // reset answers coming from the session
+            this.slide.answers = false;
```

Comment:
Same naming update here, slide.answers is not straightforward. this.slide.(user_?)session_answers or something similar would help understanding its purpose.